ui/src/app/shared/models.ts

```diff
@@ -721,3 +721,59 @@ export interface GnuPGPublicKey {
 }
 
 export interface GnuPGPublicKeyList extends ItemsList<GnuPGPublicKey> {}
+
+// https://kubernetes.io/docs/reference/kubectl/overview/#resource-types
+
+export enum ResourceKinds {
+    ANY = '*',
```

Comment:
Looks like we are using only keys of that map, but not values. The `ANY` is now supposed to be suggested to the user. Should we just store keys are an array and replace `ANY` with `*` ?

Reply:
Sounds good. The reason I created this as an enum was for type safety in a ResourceKindSelector component - I'll convert to array for now and create the type in refactor PR